nix/runtime.nix

```
@@ -131,6 +140,29 @@ rec {
             ];
           };
         };
+        kupo = {
+          service = {
+            image = "cardanosolutions/kupo:${kup.tag}";
+            volumes = [
+              "${config.cardano-configurations}/network/${config.network.name}:/config"
+              "${nodeIpcVol}:/ipc"
+              "kupo_db"
```

Also need to add `kupoDbVol = "kupo-db";` next to `nodeDbVol` and `nodeIpcVol`
and  `"${kupoDbVol}" = { };` next to `"${nodeIpcVol}" = { };`
```suggestion
              "${kupoDbVol}:/kupo-db"
```